packages/core/src/columns/timestamps.ts

```diff
@@ -81,7 +83,7 @@ export const makeTimestampsHelpers = (
     const updatedAtInjectorSnake = makeInjector(
       updatedAtRegexSnake,
       raw(`${quotedUpdatedAtSnakeCase} = ${now}`),
-      'updated_at',
+      'updatedAt',
```

Comment:
I see, so this was the mistake: columns are always in camelCase on the app side.

And the changes above are not changing the behavior.